resources/lang/vi/community.php

```diff
@@ -22,108 +22,109 @@
+            'big_description' => 'Yêu thích osu!?<br/>
+                                Hãy ủng hộ các dev trong osu! :D',
             'small_description' => '',
-            'support_button' => 'I want to support osu!',
+            'support_button' => 'Tôi muốn ủng hộ osu!',
         ],
 
-        'dev_quote' => 'osu! is a completely free-to-play game, but running it is most definitely not so free. Between the cost of commissioning servers and high quality international bandwidth, the time spent maintaining the system and community, providing prizes for competitions, answering support questions and generally keeping people happy, osu! consumes quite a hefty amount of money! Oh, and don\'t forget the fact that we do it without any advertising or partnering with silly toolbars and the likes!
-            <br/><br/>osu! is at the end of the day largely run by myself, to which you may know best as "peppy".
-            I have had to quit my day job in order to keep up with osu!,
-            and do at times struggle to maintain the standards I strive for.
-            I would like to offer my personal thanks to those who have supported osu! thus far,
-            and just as much to those who continue to support this amazing game and community into the future :).',
+        'dev_quote' => 'osu! là một trò chơi hoàn toàn miễn phí, nhưng để cho osu chạy được thì lại không hề miễn phí. Giữa chi phí cho các máy chủ hoạt động và băng thông quốc tế chất lượng cao, thời gian dành ra để bảo trì hệ thống và cộng đồng osu!, việc cung cấp giải thưởng cho các cuộc thi, việc trả lời các câu hỏi giúp đỡ và giữ được sự vui vẻ cho mọi người như thường lệ, số lượng tiền mà osu tiêu tốn khá là khổng lồ. À, và chưa kể chúng tôi làm mà không nhờ đến sự quảng cáo nào hay hợp tác với nhưng thanh công cụ vớ vẫn và tương tự!
+            <br/><br/>Suy cho cùng thì osu chạy được là phần lớn nhờ vào tôi, người mà bạn có thể biết đến với cái tên "peppy".
+            Tồi đã phải nghỉ công việc của mình để mà bắt kịp với osu!,
+            và cũng lúc cố gắng để duy trì những tiêu chuẩn mà tôi đặt ra.
+            Tôi muốn biếu tặng trực tiếp sự cảm tạ của tôi đối những người đã ủng hộ osu!,
+            và cũng như những người đang hỗ trợ trò chơi tuyệt với này cùng với cộng đồng của nó trong tương lai :).',
 
         'why_support' => [
-            'title' => 'Why should I support osu!?',
+            'title' => 'Tại sao tôi nên ủng hộ osu!?',
             'blocks' => [
-                'dev' => 'Developed and maintained predominantly by one guy in Australia',
-                'time' => 'Takes so much time to keep running that it\'s no longer possible to call it a "hobby".',
-                'ads' => 'No ads anywhere. <br/><br/>
+                'dev' => 'Được phát triển và duy trì một cách vượt trội bởi một người sống ở Úc',
+                'time' => 'Dành quá nhiều thời gian để trò chơi chạy được đến mức không còn gọi nó là một "thú vui" nữa.',
```

Comment:
`Dành quá nhiều thời gian để giữ osu!/nó hoạt động` would be better

Reply:
**Tốn quá nhiều thời gian để giữ cho trò chơi hoạt động, đến mức không thể gọi đây là một "thú vui" thêm nữa.**
Zdai thấy chữ "tốn" nó khốn khổ hơn chữ "dành". Với lại lần sau nhớ để ý mấy chữ như `possible` để dịch khỏi bị thiếu.

Reply:
Mình dịch theo hướng neutral, và cũng có thêm `possible` rồi